### ISSUE TITLE
Form fields error messages are not flashed when validating 

### ISSUE THREAD

**Issue by tchapi** [area17/twill#605]
Hi,

On a fresh install of Laravel with Twill 2, when I use rules to validate form fields entry, the error messages are not shown on the relevant field, only a _"Your submission could not be validated"_ global message appears.

<img width="977" alt="Screenshot 2020-03-23 at 13 13 29" src="https://user-images.githubusercontent.com/1944007/77315670-3b489580-6d08-11ea-8a59-c696b4d46edf.png">

Here are the relevant parts :

`composer.json` 
```
"require": {
        "php": "^7.2.5",
        "area17/twill": "^2.0",
        "fideloper/proxy": "^4.2",
        "fruitcake/laravel-cors": "^1.0",
        "guzzlehttp/guzzle": "^6.3",
        "laravel/framework": "^7.0",
        "laravel/tinker": "^2.0"
    },
```

**Installation process I underwent, in a nutshell:**

```
composer create-project --prefer-dist laravel/laravel blog
composer require area17/twill:"^2.0"
php artisan twill:install
php artisan twill:module article
php artisan migrate
```

**Only modifications :** 

`ArticleRequest.php` : 
```
class ArticleRequest extends Request
{
    public function rules()
    {
        return [
          'description' => 'required|alpha_dash',
        ];
    }
}
```

`twill-navigation.php`: 
```
return [
  'articles' => [
      'title' => 'Articles',
      'module' => true
  ]
];
```

`routes/admin.php`: 
```
Route::module('articles');
```
No other modifications whatsoever.

I then proceed on creating a new 'article', and when I edit it, I put some non-valid characters in the description, I only get the global message.

**What I expected**

If I'm not mistaken, the field should be flagged as non-valid, and be at least red-bordered, with a message (_"The description may only contain letters, numbers, dashes and underscores."_) attached to it.

As far as I understand, the response from the backend / api is correct, with the error message and relevant field : 

<img width="1021" alt="Screenshot 2020-03-23 at 13 16 28" src="https://user-images.githubusercontent.com/1944007/77315880-a2664a00-6d08-11ea-973b-b84618d8f46b.png">

Is there something that I missed ? It looks like a bug to me.
Happy to give more details if needed.

Thanks a lot !

**Comment by arkid** [comment]
Just found the same thing, I'm sure there is a problem here.    I'm on sites that were built in 1.2 and upgrade to v2 recently.    Field validation that used to work now doesn't properly.   

No matter how simple the model, adding just the most simple 'required' requirement to say a text field doesn't result in 1) anything happening near the problem field to flag it, 2) a custom msg appearing at the bottom of the screen.

@ifox any chance of clarifying what's going on?    For me this is really important, I have a site I need to put live this week and if field validation doesn't work I think I'm going to have a lot of problems for one of my more complex models with lots of fields.

**Comment by arkid** [comment]
Also just to say, when you try and use the preview option when editing content you get the same generic validation error if there is a problem calling the blade template used for previewing.   

This may be a totally separate thing or this generic error that now appears all the time is catching a more accurate error that normally shows ?

**Comment by philippdaun** [comment]
Can confirm this behavior after updating to the latest release (2.0.1). No validation messages on the inputs themselves, just the flash bar. Worked fine before upgrading from a 2.0 pre-release version.

Could it be a compatibility issue with Laravel 6?

**Comment by arkid** [comment]
Don't think it's Laravel 6 specific @philippdaun , i'm on Laravel 5.8 for a few projects still and get the same issue.

**Comment by philippdaun** [comment]
@arkid What's strange is I looked through all the commits between when it was last working for me and the latest release and there's nothing that looks probable to be causing this. That's why I figured it might be further up the tree. But yeah, looks like it's not a Laravel regression.

**Comment by ifox** [comment]
Hi guys, I'm investigating this and planning to release 2.1 asap with recent bug fixes and exciting PRs I'm reviewing.

**Comment by arkid** [comment]
Amazing thanks @ifox ! :)

**Comment by tchapi** [comment]
Hi @ifox 
Do you think you can release a patch version (2.0.2?) with this fix, sometimes soon, before the 2.1 ? We have migrated a few clients to Twill 2.0 but now we're stuck with this bug which is quite annoying

Thanks a lot !

**Comment by tchapi** [comment]
Ok, in fact, I did some more testing following https://github.com/area17/twill/commit/fc5b16a9179fcba57d3424ef2456a9efa74ce68f.

The fields are now correctly highlighted as errored (in red) as expected, but the `required` fields do not have any text message:

<img width="1440" alt="Screenshot 2020-04-21 at 16 08 40" src="https://user-images.githubusercontent.com/1944007/79875949-b033eb80-83ea-11ea-8e92-e45883f49161.png">

(In this example, both fields have the `required|alpha_num` requirement)

The return from the API is good though, with the correct error messages:

<img width="568" alt="Screenshot 2020-04-21 at 16 24 31" src="https://user-images.githubusercontent.com/1944007/79877550-a57a5600-83ec-11ea-838d-3a2cb1a3900c.png">

The culprit is in `mixins/inputFrame.js` here : https://github.com/area17/twill/blob/4b61b78e1ab746fbe9975be23993adfd4a6d1391/frontend/js/mixins/inputFrame.js#L58

I don't really understand why you would silence the messages finishing with 'is required.' — that is the bug here. More over, this is quite dangerous since now the UI is localized ...

Do I need to create a separate issue for that ?
